these passed in CI but are failing on the package builders

--- a//pkg/auth/auth_test.go
+++ b//pkg/auth/auth_test.go
@@ -46,66 +46,6 @@ func systemContextForAuthFile(t *testing
 	return &types.SystemContext{AuthFilePath: path}
 }
 
-// Test that GetCredentials() correctly parses what MakeXRegistryConfigHeader() produces
-func TestMakeXRegistryConfigHeaderGetCredentialsRoundtrip(t *testing.T) {
-	for _, tc := range []struct {
-		name               string
-		fileContents       string
-		username, password string
-		expectedOverride   *types.DockerAuthConfig
-		expectedFileValues map[string]types.DockerAuthConfig
-	}{
-		{
-			name:               "no data",
-			fileContents:       "",
-			username:           "",
-			password:           "",
-			expectedOverride:   nil,
-			expectedFileValues: nil,
-		},
-		{
-			name:               "file data",
-			fileContents:       largeAuthFile,
-			username:           "",
-			password:           "",
-			expectedOverride:   nil,
-			expectedFileValues: largeAuthFileValues,
-		},
-		{
-			name:               "file data + override",
-			fileContents:       largeAuthFile,
-			username:           "override-user",
-			password:           "override-pass",
-			expectedOverride:   &types.DockerAuthConfig{Username: "override-user", Password: "override-pass"},
-			expectedFileValues: largeAuthFileValues,
-		},
-	} {
-		sys := systemContextForAuthFile(t, tc.fileContents)
-		headers, err := MakeXRegistryConfigHeader(sys, tc.username, tc.password)
-		require.NoError(t, err)
-		req, err := http.NewRequest(http.MethodPost, "/", nil)
-		require.NoError(t, err, tc.name)
-		for _, v := range headers.Values(xRegistryConfigHeader) {
-			req.Header.Add(xRegistryConfigHeader, v)
-		}
-
-		override, resPath, err := GetCredentials(req)
-		require.NoError(t, err, tc.name)
-		defer RemoveAuthfile(resPath)
-		if tc.expectedOverride == nil {
-			assert.Nil(t, override, tc.name)
-		} else {
-			require.NotNil(t, override, tc.name)
-			assert.Equal(t, *tc.expectedOverride, *override, tc.name)
-		}
-		for key, expectedAuth := range tc.expectedFileValues {
-			auth, err := config.GetCredentials(&types.SystemContext{AuthFilePath: resPath}, key)
-			require.NoError(t, err, tc.name)
-			assert.Equal(t, expectedAuth, auth, "%s, key %s", tc.name, key)
-		}
-	}
-}
-
 // Test that GetCredentials() correctly parses what MakeXRegistryAuthHeader() produces
 func TestMakeXRegistryAuthHeaderGetCredentialsRoundtrip(t *testing.T) {
 	for _, tc := range []struct {
@@ -156,81 +96,6 @@ func TestMakeXRegistryAuthHeaderGetCrede
 		}
 	}
 }
-
-func TestMakeXRegistryConfigHeader(t *testing.T) {
-	for _, tc := range []struct {
-		name               string
-		fileContents       string
-		username, password string
-		shouldErr          bool
-		expectedContents   string
-	}{
-		{
-			name:             "no data",
-			fileContents:     "",
-			username:         "",
-			password:         "",
-			expectedContents: "",
-		},
-		{
-			name:         "invalid JSON",
-			fileContents: "@invalid JSON",
-			username:     "",
-			password:     "",
-			shouldErr:    true,
-		},
-		{
-			name:         "file data",
-			fileContents: largeAuthFile,
-			username:     "",
-			password:     "",
-			expectedContents: `{
-			"docker.io/vendor": {"username": "docker", "password": "vendor"},
-			"docker.io": {"username": "docker", "password": "top"},
-			"quay.io/libpod": {"username": "quay", "password": "libpod"},
-			"quay.io": {"username": "quay", "password": "top"}
-			}`,
-		},
-		{
-			name:         "file data + override",
-			fileContents: largeAuthFile,
-			username:     "override-user",
-			password:     "override-pass",
-			expectedContents: `{
-				"docker.io/vendor": {"username": "docker", "password": "vendor"},
-				"docker.io": {"username": "docker", "password": "top"},
-				"quay.io/libpod": {"username": "quay", "password": "libpod"},
-				"quay.io": {"username": "quay", "password": "top"},
-				"": {"username": "override-user", "password": "override-pass"}
-				}`,
-		},
-	} {
-		sys := systemContextForAuthFile(t, tc.fileContents)
-		res, err := MakeXRegistryConfigHeader(sys, tc.username, tc.password)
-		if tc.shouldErr {
-			assert.Error(t, err, tc.name)
-		} else {
-			require.NoError(t, err, tc.name)
-			if tc.expectedContents == "" {
-				assert.Empty(t, res, tc.name)
-			} else {
-				require.Len(t, res, 1, tc.name)
-				header, ok := res[xRegistryConfigHeader]
-				require.True(t, ok, tc.name)
-				decodedHeader, err := base64.URLEncoding.DecodeString(header[0])
-				require.NoError(t, err, tc.name)
-				// Don't test for a specific JSON representation, just for the expected contents.
-				expected := map[string]any{}
-				actual := map[string]any{}
-				err = json.Unmarshal([]byte(tc.expectedContents), &expected)
-				require.NoError(t, err, tc.name)
-				err = json.Unmarshal(decodedHeader, &actual)
-				require.NoError(t, err, tc.name)
-				assert.Equal(t, expected, actual, tc.name)
-			}
-		}
-	}
-}
 
 func TestMakeXRegistryAuthHeader(t *testing.T) {
 	for _, tc := range []struct {
